### PR TITLE
Update Functionality Completed

### PR DESCRIPTION
**Pull request by RichDTran**

### CODE REVIEW

File: src/pages/Home.js [hunk 1]
```diff
@@ -4,8 +4,19 @@ const Home = () => {
...
+      <h1>Meet the Villains!</h1>
+      <br>
+
+      </br>
+      <p>Sinister Bachelors & Bachelorettes</p>
+
+      <p> What kind of villain are you most attracted to?
+{/*         
+          <img alt= "Devil Emoji" 
+          src="https://cdn.pixabay.com/photo/2020/12/24/01/14/smile-5856241_1280.png"> </img> */}
```
Comment (elysemontano) [inline src/pages/Home.js]: Do you need this image?  If so, please comment it back in, if not, please remove it.

File: src/pages/VillainShow.js [hunk 2]
```diff
@@ -1,12 +1,13 @@
...
-import { useParams } from "react-router-dom"
-import {Card, CardBody, CardSubtitle, CardText, CardTitle} from 'reactstrap'
+import {Card, CardBody, CardSubtitle, CardText, CardTitle, Button} from 'reactstrap'
+import { NavLink, useParams } from "react-router-dom"
+// import {currentVillain} from "./src/pages/VillainEdit"
 
 const VillainShow = ({ villains }) => {
   const { id } = useParams()
   let selectedVillain = villains.find((villain) => villain.id === +id)
+  let currentVillain = villains?.find((villain) => villain.id === +id)
```
Comment (elysemontano) [inline src/pages/VillainShow.js]: This looks the same as line 8, do you need both of these?

**Review by elysemontano — CHANGES_REQUESTED** [review]
This is looking excellent!  A few small things to look at.  
Consider adding the value attribute on your input tags that has the value of the current state.  This will allow the current value to be your placeholder so the user knows that villain they are updating.
I left a few other comments to address as well.

File: src/pages/VillainShow.js [hunk 3]
```diff
@@ -1,12 +1,13 @@
 import React from "react"
-import { useParams } from "react-router-dom"
-import {Card, CardBody, CardSubtitle, CardText, CardTitle} from 'reactstrap'
+import {Card, CardBody, CardSubtitle, CardText, CardTitle, Button} from 'reactstrap'
+import { NavLink, useParams } from "react-router-dom"
+// import {currentVillain} from "./src/pages/VillainEdit"
```
Comment (elysemontano) [inline src/pages/VillainShow.js]: Doesn't look like you are using this import anymore.  Can you remove it?

**Review by elysemontano — CHANGES_REQUESTED** [review]
There are a few things to either uncomment out or delete.

File: src/pages/Home.js (hunk 1 above)
Comment (RichDTran) [reply src/pages/Home.js]: Plan to use this image.

File: src/pages/VillainShow.js (hunk 2 above)
Comment (RichDTran) [reply src/pages/VillainShow.js]: deleted

File: src/pages/VillainShow.js (hunk 3 above)
Comment (RichDTran) [reply src/pages/VillainShow.js]: Removed

File: src/pages/Home.js (hunk 1 above)
Comment (elysemontano) [reply src/pages/Home.js]: Best practice is to leave main clean of commented-out code unless it is for documentation.  If you plan to use it, it is best to either implement it now by uncommenting or adding it when you plan on using it.

**Review by elysemontano — CHANGES_REQUESTED** [review]
Best practice is to leave main clean of commented-out code unless it is for documentation. If you plan to use it, it is best to either implement it now by uncommenting or adding it when you plan on using it.

File: src/components/Header.js [hunk 4]
```diff
@@ -22,6 +22,27 @@ const Header = () => {
...
+        <span class="drop"></span>
+        <span class="drop"></span>
+        <span class="drop"></span>
+        <span class="drop"></span>
+        <span class="drop"></span>
+        <span class="drop"></span>
+        <span class="drop"></span>
+        <span class="drop"></span>
+        <span class="drop"></span>
+        <span class="drop"></span>
```
Comment (elysemontano) [inline src/components/Header.js:45]: Not a deal breaker for this PR, but this is a lot of span tags.

**Review by elysemontano — APPROVED** [review]
Left a comment in Header to look at before merging

**Pull request merged**